frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -190,7 +192,7 @@ export default function InteractiveWaveformModal({
     // only available for versions greater than 0.25.2
     const compatibleVersions = pulse3dVersions.filter((v) => semverGte(v, "0.25.2"));
     setFilteredVersions([...compatibleVersions]);
-    if (numberOfJobsInUpload > 2) {
+    if (usageQuota && usageQuota.limits && numberOfJobsInUpload > 2 && usageQuota.limits.jobs !== -1) {
```

Comment:
Is there always going to be `usageQuota.limits.jobs` if `usageQuota.limits` exists?